src/components/SpiceList.js

```diff
@@ -20,7 +28,7 @@ class SpiceList extends React.Component {
   render() {
     return (
       <section className="spice-list">
-        <Filter search={this.state.search} fourStarOnly={this.state.fourStarOnly} />
+        <Filter search={this.state.search} fourStarOnly={this.state.fourStarOnly} changeSearch={ this.changeSearch } changeFourStar={ this.changeFourStar }/>
```

Comment:
One thing that helps with readability for components that take in a lot of props is to separate out each prop on its own line: 

```jsx
<Filter 
    search={this.state.search} 
    fourStarOnly={this.state.fourStarOnly} 
    changeSearch={ this.changeSearch } 
    changeFourStar={ this.changeFourStar }
/>
```